### ISSUE TITLE
MONGODB_HOST variable doesn't seem to be respected

### ISSUE THREAD

**Issue by drzaiusx11** [astzweig/docker-wildduck#3]
mongo seems to startup fine:
```
mongo_1  | 2019-01-24T18:09:33.870+0000 I CONTROL  [main] Automatically disabling TLS 1.0, to force-enable TLS 1.0 specify --sslDisabledProtocols 'none'
mongo_1  | 2019-01-24T18:09:33.872+0000 I CONTROL  [initandlisten] MongoDB starting : pid=1 port=27017 dbpath=/data/db 64-bit host=c2ce19956a7a
mongo_1  | 2019-01-24T18:09:33.872+0000 I CONTROL  [initandlisten] db version v4.0.5
mongo_1  | 2019-01-24T18:09:33.872+0000 I CONTROL  [initandlisten] git version: 3739429dd92b92d1b0ab120911a23d50bf03c412
mongo_1  | 2019-01-24T18:09:33.872+0000 I CONTROL  [initandlisten] OpenSSL version: OpenSSL 1.0.2g  1 Mar 2016
mongo_1  | 2019-01-24T18:09:33.872+0000 I CONTROL  [initandlisten] allocator: tcmalloc
mongo_1  | 2019-01-24T18:09:33.872+0000 I CONTROL  [initandlisten] modules: none
mongo_1  | 2019-01-24T18:09:33.872+0000 I CONTROL  [initandlisten] build environment:
mongo_1  | 2019-01-24T18:09:33.872+0000 I CONTROL  [initandlisten]     distmod: ubuntu1604
mongo_1  | 2019-01-24T18:09:33.872+0000 I CONTROL  [initandlisten]     distarch: x86_64
mongo_1  | 2019-01-24T18:09:33.872+0000 I CONTROL  [initandlisten]     target_arch: x86_64
mongo_1  | 2019-01-24T18:09:33.872+0000 I CONTROL  [initandlisten] options: { net: { bindIpAll: true } }
mongo_1  | 2019-01-24T18:09:33.872+0000 W STORAGE  [initandlisten] Detected unclean shutdown - /data/db/mongod.lock is not empty.
mongo_1  | 2019-01-24T18:09:33.873+0000 I STORAGE  [initandlisten] Detected data files in /data/db created by the 'wiredTiger' storage engine, so setting the active storage engine to 'wiredTiger'.
```
...
```
mongo_1  | 2019-01-24T17:59:07.461+0000 I NETWORK  [initandlisten] waiting for connections on port 27017
```

but then when mail_1 boots its unable to connect. I have tried setting MONGODB_HOST explicitly to mongodb://mongo:27017/wildduck

Error:
```
mail_1   | ERR! Db Failed to setup database connection
mail_1   | { short_message: 'WILDDUCK [Exception] failed to connect to server [127.0.0.1:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]',
mail_1   |   full_message: 'MongoNetworkError: failed to connect to server [127.0.0.1:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]\n    at Pool.<anonymous> (/var/nodemailer/wildduck/node_modules/mongodb-core/lib/topologies/server.js:564:11)\n    at emitOne (events.js:116:13)\n    at Pool.emit (events.js:211:7)\n    at Connection.<anonymous> (/var/nodemailer/wildduck/node_modules/mongodb-core/lib/connection/pool.js:317:12)\n    at Object.onceWrapper (events.js:317:30)\n    at emitTwo (events.js:126:13)\n    at Connection.emit (events.js:214:7)\n    at Socket.<anonymous> (/var/nodemailer/wildduck/node_modules/mongodb-core/lib/connection/connection.js:246:50)\n    at Object.onceWrapper (events.js:315:30)\n    at emitOne (events.js:116:13)',
mail_1   |   _exception: 'yes',
mail_1   |   _error: 'failed to connect to server [127.0.0.1:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]',
mail_1   |   _name: 'MongoNetworkError',
mail_1   |   facility: 'wildduck',
mail_1   |   host: '83dc939ba677',
mail_1   |   timestamp: 1548352769.84,
mail_1   |   _component: 'wildduck' }
```

docker-compose version 1.21.1, build 5a3f1a3
docker version 18.03.1-ce, build 9ee9f40
os: osx 10.12.6

**Comment by Alwinius** [comment]
same for me on clean Ubuntu 18.04 Docker version 18.09.1.

**Comment by drzaiusx11** [comment]
NOTE: this is for the image I built from running docker-compose.yml in the root of the project

**Comment by Alwinius** [comment]
the default hostname set in wildduck is `mongodb` now, but the database container will have the default hostname `mongo`

**Comment by rqelibari** [comment]
> the default hostname set in wildduck is `mongodb` now, but the database container will have the default hostname `mongo`

We will modify the default container names with the next release. For right now, just use the `MONGODB_HOST` variable and set it to `mongodb://mongo:27017/wildduck` or rename the `mongo`container to `mongodb` in the `docker-compose.yml`.


I just fixed that with [f4ed6b2](https://github.com/astzweig/docker-wildduck/commit/f4ed6b2b76b0dce485d1e231dc1ca71c582c56d9). So no need to do anything anymore.

EDIT: Add reference to fixing commit.
